### PR TITLE
UICIRC-698: Create new setting for title level requests

### PR DESCRIPTION
**Pull request by Stanislau-Snapok**
## Purpose
Create new setting for title level requests

## Approach
In scope of this task new settings was added. New components fully covered with RTL/Jest tests.

## Refs
https://issues.folio.org/browse/UICIRC-698

## Screenshots
image
image

### CODE REVIEW

**Review by Dmitriy-Litvinenko — CHANGES_REQUESTED** [review]

File: src/constants.js [hunk 1]
```diff
@@ -469,3 +469,27 @@ export const MAX_UNPAGED_RESOURCE_COUNT = 1000;
 export default '';
 
 export const DATE_FORMAT = 'YYYY/MM/DD';
+
+export const TITLE_LEVEL_REQUESTS_PROPS_IDS = {
```
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply src/constants.js]: done

File: src/settings/TitleLevelRequests/NoticeTemplates/NoticeTemplates.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+const NoticeTemplates = (props) => {
+  const {
+    intl: {
+      formatMessage,
+    },
+    templates,
+  } = props;
+  const valuesForChoise = [
+    { value: 'null', label: 'Select notice' },
```
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply src/settings/TitleLevelRequests/NoticeTemplates/NoticeTemplates.js]: done

File: src/settings/TitleLevelRequests/NoticeTemplates/NoticeTemplates.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import { TITLE_LEVEL_REQUESTS_PROPS_IDS } from '../../../constants';
+
+const NoticeTemplates = (props) => {
+  const {
+    intl: {
+      formatMessage,
+    },
+    templates,
+  } = props;
+  const valuesForChoise = [
```
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply src/settings/TitleLevelRequests/NoticeTemplates/NoticeTemplates.js]: done

File: src/settings/TitleLevelRequests/utils/getInitialValues.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  head,
+} from 'lodash';
+import { TITLE_LEVEL_REQUESTS_FORM_DEFAULT_PROPS } from '../../../constants';
+
+const getInitialValues = (settings) => {
+  let config;
+  const value = isEmpty(settings) ? '' : head(settings).value;
+
+  try {
+    config = { ...TITLE_LEVEL_REQUESTS_FORM_DEFAULT_PROPS, ...JSON.parse(value) };
```
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply src/settings/TitleLevelRequests/utils/getInitialValues.js]: refactored

**Pull request merged**